Memorize/MemoryGame.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  MemoryGame.swift
+//  Memorize
+//
+//  Created by Kedar Dhere on 8/10/22.
+//
+
+import Foundation
+
+struct MemoryGame<CardContent> where CardContent: Equatable {
+
+    // MARK: - Properties
+    struct Card: Identifiable {
+
+        var isFaceUp: Bool = false
```

Comment:
Here and throughout, I would move the line skip.  So, instead of:
```
    // MARK: - Properties
    struct Card: Identifiable {

        var isFaceUp: Bool = false
```
I would write
```
    // MARK: - Properties

    struct Card: Identifiable {
        var isFaceUp: Bool = false
```

Reply:
The requested changes are completed.